inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/source/pulsar/PulsarSourceRequest.java

```diff
@@ -76,6 +76,12 @@ public class PulsarSourceRequest extends SourceRequest {
             + " Available options are earliest, latest, external-subscription, and specific-offsets.")
     private String scanStartupMode = "earliest";
 
+    @ApiModelProperty("Configure the Pulsar subscription mode. Available options are Exclusive, Shared, and Failover.")
```

Comment:
What about the Key_Shared subscription type?

Reply:
We may no longer need users to specify subscription types.